blog/config/settings.yml

```
@@ -5,3 +5,8 @@ link:
   contact: "https://www.railstutorial.org/contact"
 text:
   title: "Ruby on Rails Tutorial Sample App"
+  string_validate: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+number:
+  maxName: 50
```

đặt tên dưới dạng snake_case e nhé
max_name

nên đặt kiểu ntn e ơi
```suggestion
user:
     max_length:
           name: 50
           
```

---

tương tự các cái khác

---

vang a